front-end/src/styles/analyzersAndSeriesStyles.css

```diff
@@ -4,6 +4,13 @@
   justify-content: space-around;
 }
 
+.columnHeaders2 {
```

Comment:
Why name is so bad please dont do this, this is also code that will be rewritten but cmon